### PR TITLE
add One Million Posts Corpus

### PR DESCRIPTION
**Pull request by aseifert**
- **Name:** One Million Posts Corpus
- **Description:** The “One Million Posts” corpus is an annotated data set consisting of user comments posted to an Austrian newspaper website (in German language).
- **Paper:** https://dl.acm.org/doi/10.1145/3077136.3080711
- **Data:** https://github.com/OFAI/million-post-corpus
- **Motivation:** Big German (real-life) dataset containing different annotations around forum moderation with expert annotations.

### Checkbox

- [X] Create the dataset script `/datasets/my_dataset/my_dataset.py` using the template
- [X] Fill the `_DESCRIPTION` and `_CITATION` variables
- [X] Implement `_infos()`, `_split_generators()` and `_generate_examples()`
- [X] Make sure that the `BUILDER_CONFIGS` class attribute is filled with the different configurations of the dataset and that the `BUILDER_CONFIG_CLASS` is specified if there is a custom config class.
- [X] Generate the metadata file `dataset_infos.json` for all configurations
- [X] Generate the dummy data `dummy_data.zip` files to have the dataset script tested and that they don't weigh too much (<50KB)
- [X] Add the dataset card `README.md` using the template : fill the tags and the various paragraphs
- [X] Both tests for the real data and the dummy data pass.

### CODE REVIEW

File: datasets/omp/omp.py [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+        datasets.BuilderConfig(
+            name="posts_labeled", version=VERSION, description="This part of my dataset covers a first domain"
+        ),
+        datasets.BuilderConfig(
+            name="posts_unlabeled", version=VERSION, description="This part of my dataset covers a second domain"
+        ),
+        datasets.BuilderConfig(
+            name="articles", version=VERSION, description="This part of my dataset covers a second domain"
+        ),
+    ]
```
Comment (lhoestq) [inline datasets/omp/omp.py:134]: can you give proper description of the different configs please ?

**Review by lhoestq — COMMENTED** [review]
really cool thanks :) 

I left one comment

File: datasets/omp/omp.py (hunk 1 above)
Comment (aseifert) [reply datasets/omp/omp.py:134]: oops, forgot about that! will do

Comment (aseifert) [reply datasets/omp/omp.py]: oops, fixed!

**Review by lhoestq — APPROVED** [review]
Awesome thank you :)

**Comment by lhoestq** [comment]
merging since the CI is fixed on master

**Pull request merged**